docs/source/common/trainer.rst

```diff
@@ -669,43 +669,38 @@ fast_dev_run
 
 |
 
-Runs n if set to ``n`` (int) else 1 if set to ``True`` batch(es) of train, val and test
```

Comment:
The wording here made it sound like `trainer.fit()` would still run 1 test batch. Not the case